x-pack/plugins/security_solution/public/common/components/events_tab/events_query_tab_body.test.tsx

```diff
@@ -106,7 +106,7 @@ describe('EventsQueryTabBody', () => {
     );
 
     expect(queryByText('MockedStatefulEventsViewer')).toBeInTheDocument();
-    expect(mockGetDefaultControlColumn).toHaveBeenCalledWith(4);
+    expect(mockGetDefaultControlColumn).toHaveBeenCalledWith(5);
```

Comment:
is there test coverage for counts when the flag is on (disabled)? thought so but if not should probably add a case here instead of changing the values. wdyt?

Reply:
good point, I was a bit lazy and only modified the existing tests... I just added 2 more tests to check all scenarios, thanks!